cmd/image-builder/config_test.go

```diff
@@ -143,3 +143,176 @@ func Test_getVariants(t *testing.T) {
 		})
 	}
 }
+
+func TestLoadGitStateConfigFromEnv(t *testing.T) {
```

Comment:
Test is missing scenario for unsupported github event. We should make sure image-builder will not run for unsupported events.

Reply:
Added for both github and prow